### PR TITLE
Btn hovers added and tiny modification to html

### PR DESCRIPTION
**Pull request by KingLyrics**
I added an ID to specifically identify facebook logo... I could have used another way may with psuedo but i kept missing it. Using first of child or so on. If you feel you want us to change it that would be very okay. I am still learning after all. But the hovers are working quite well.

### CODE REVIEW

**Comment by KingLyrics** [comment]
Been battling the facebook hover effect for a while now. Your help would be very much appreciated :)

File: src/sass/main.scss [hunk 1]
```diff
@@ -213,9 +213,18 @@ body {
...
 
+	&:hover{
+		background-color: $clr-primary-400;
+	}
+
 	&--alternate {
 		background-color: $clr-neutral-100;
 		color: $clr-neutral-800;
+
+		&:hover{
```
Comment (gmagnenat) [inline src/sass/main.scss]: cool 😎 you can add a little transition for the hover color. Makes it more smooth

File: src/sass/main.scss [hunk 2]
```diff
@@ -213,9 +213,18 @@ body {
...
+			background-color: $clr-tertiary-400;
+			color: $clr-neutral-100;
```
Comment (gmagnenat) [inline src/sass/main.scss]: the color of the text here should be neutral-800 (dark)

File: src/sass/main.scss [hunk 3]
```diff
@@ -258,9 +265,12 @@ body {
 
 	.socials {
 		margin-top: 4rem;
-		// padding-bottom: 2rem;
 		display: flex;
 		gap: 1.2rem;
 		align-items: center;
+
+		#facebook:hover{
```
Comment (gmagnenat) [inline src/sass/main.scss]: By the design, I think all links have the same color on hover. So you don't need unique ID here.
Try to use a css filter instead.

**Review by gmagnenat — CHANGES_REQUESTED** [review]

File: src/sass/main.scss (hunk 1 above)
Comment (KingLyrics) [reply src/sass/main.scss]: Done and dusted

File: src/sass/main.scss (hunk 2 above)
Comment (KingLyrics) [reply src/sass/main.scss]: Corrected

File: src/sass/main.scss (hunk 3 above)
Comment (KingLyrics) [reply src/sass/main.scss]: Okay, a little unsure about how to go about this one. Not familiar with the filter property.

Comment (gmagnenat) [reply src/sass/main.scss]: @KingLyrics I added the hover rule with filter. You can check it out in the last commit on this branch

Comment (gmagnenat) [reply src/sass/main.scss]: You can check tools like this to help find the value. https://isotropic.co/tool/hex-color-to-css-filter/

**Review by gmagnenat — APPROVED** [review]
All good !

**Pull request merged**